js/index.js

```diff
@@ -106,3 +101,50 @@ messageForm.addEventListener('submit', function(event) {
      event.target.reset();
 
 });
+
+///lesson-15
+
+
+// 'MY_GITHUB_USERNAME'
+const myUsername = 'Lukinskai';
+
+// Make a GET request to fetch repository 
```

Comment:
Same as above - most likely the person reading your code will have some understanding of this code and wouldn't need this comment. Keeps the code clean.